### ISSUE TITLE
Is the pre-trained model uploaded the same that achieved the results in the paper?

### ISSUE THREAD

**Issue by fjzs** [jiaxue1993/pytorch-material-classification#5]
Thanks

**Comment by jiaxue-ai** [comment]
I've adapted the code to PyTorch, which may introduce some training discrepancies. But you should still achieve comparable classification accuracy.
